### PR TITLE
Fixed order status bulk update

### PR DESCRIPTION
**Pull request by antho-girard**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | This PR fixes order status bulk update in Admin Orders |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-974 |
| How to test? | In BO > Admin Orders, select at least two orders and change orders status using bulk action button. |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: controllers/admin/AdminOrdersController.php [hunk 1]
```diff
@@ -386,7 +386,7 @@ public function processBulkUpdateOrderStatus()
     {
         if (Tools::isSubmit('submitUpdateOrderStatus')
             && ($id_order_state = (int)Tools::getValue('id_order_state'))) {
-            if ($this->access('edit') !== '1') {
+            if (true != $this->access('edit')) {
```
Comment (mickaelandrieu) [inline controllers/admin/AdminOrdersController.php]: `!==` as ``access`` return a boolean :)

**Comment by vincentbz** [comment]
status : 'QA approved' 
thanks Anthony

**Comment by mickaelandrieu** [comment]
ping @antho-girard this just need a little update in order to be merged :)

**Comment by antho-girard** [comment]
Hi @mickaelandrieu,

Thanks for your feedback !
Title and operator are ok :)

**Comment by mickaelandrieu** [comment]
I dont know why but tests are broken.. would you mind to rebase please ?

Maybe my suggestion was bad ^^

**Comment by mickaelandrieu** [comment]
Thank you Anthony !

**Pull request merged**